### PR TITLE
add component homepage

### PR DESCRIPTION
**Pull request by SebSamPro**

### CODE REVIEW

**Review by Rayane0823 — APPROVED** [review]
Reviewed

**Review by LucasMerino10 — APPROVED** [review]
good!

File: frontend/src/App.jsx
Comment (Yoann-Fortin) [inline frontend/src/App.jsx:1]: File names are also capitalized.

File: frontend/src/components/home-folder/home.css
Comment (Yoann-Fortin) [inline frontend/src/components/home-folder/home.css:1]: need explanations of several values in this code

File: frontend/src/components/home-folder/home.jsx
Comment (Yoann-Fortin) [inline frontend/src/components/home-folder/home.jsx:1]: Why a href in img tag ?

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**